### PR TITLE
Update use-windows-defender-application-control-with-intelligent-secu…

### PR DESCRIPTION
**Pull request by lindspea**
…rity-graph.md

https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3518
Added note.

### CODE REVIEW

**Comment by illfated** [comment]
👍 LGTM™

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-with-intelligent-security-graph.md [hunk 1]
```diff
@@ -98,3 +98,6 @@ Modern apps are not supported with the ISG heuristic and will need to be separat
 The ISG heuristic does not authorize kernel mode drivers. The WDAC policy must have rules that allow the necessary drivers to run.  
 
 In some cases, the code integrity logs where WDAC errors and warnings are written will contain error events for native images generated for .NET assemblies. Typically, the error is functionally benign as a blocked native image will result in the corresponding assembly being re-interpreted. Review for functionality and performance for the related applications using the native images maybe necessary in some cases. 
+
+>[!NOTE]
+> A rule that explicitly allows an application will take precedence over the ISG rule that does not allow it. In this scenario, this policy is not compatible with Intune, where there is no option to add rules to the template that enables ISG. In almost any circumstance, you would need to build a custom WDAC policy, including ISG if desired.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-with-intelligent-security-graph.md]:
```suggestion
> A rule that explicitly allows an application will take precedence over the ISG rule that does not allow it. In this scenario, this policy is not compatible with Intune, where there is no option to add rules to the template that enables ISG. In most circumstances you would need to build a custom WDAC policy, including ISG if desired.
```

**Comment by e0i** [comment]
@dansimp This PR that has passed copy review that addresses issue #3518 is available for review. Thanks.

**Comment by e0i** [comment]
@Dansimp Following up for updates for this PR.

**Comment by Air-Git** [comment]
That's really great, thanks. It might seem a small change, but it makes a big difference to an engineer trying to work out how to implement this in Intune.

**Pull request merged**